jani_generator/src/jani_generator/scxml_helpers/scxml_to_jani.py

```diff
@@ -33,31 +33,39 @@
 
 
 def convert_scxml_root_to_jani_automaton(
-        scxml_root: ScxmlRoot, jani_automaton: JaniAutomaton, events_holder: EventsHolder
+        scxml_root: ScxmlRoot, jani_automaton: JaniAutomaton, events_holder: EventsHolder,
+        max_array_size: int
```

Comment:
I am not sure if this should be defined globally. We should think about making this per array

Reply:
Maybe we need a mix of both: ROS interfaces might contain unbounded arrays, and there we would definitely need to enforce a bound ourselves (e.g. in the Fibonacci example)